bazel-mode.el

```diff
@@ -83,9 +83,21 @@
      . 'font-lock-builtin-face)
     ;; Keywords for BUILD files are a subset of Starlark keywords. For details
     ;; see https://docs.bazel.build/versions/master/skylark/language.html.
```

Comment:
Remove the comment, because it's outdated:
- the URL is not relevant and the url to the spec is just above
- BUILD and bzl files have the same list of keywords (even if some of them are forbidden in BUILD files, they should be highlighted)

Reply:
Removed